src/Jeffijoe.MessageFormat/Jeffijoe.MessageFormat.csproj

```diff
@@ -18,6 +18,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="5.0.5" />
```

Comment:
I wonder if this is as performant as the [Roslyn internal pool](https://github.com/dotnet/roslyn/blob/main/src/Dependencies/PooledObjects/ObjectPool%601.cs) 🤔